### ISSUE TITLE
Tool tip is missing for "create new codespace" and "select plan" button.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode#112705]
["Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11yWCAG2.1; #A11yTCS; #Win32; #DesktopApp; #A11y_VSCode;#WCAG1.1.1; #A11ySev2;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;#Keyboard

### Environment Details:​
Operating System: Win10​
Version: 1.47.0-insider (user setup)
Commit: d5e9aa0
Date: 2020-07-08T10:52:52.439Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041
### 
Additional Details:
MAS References:WCAG1.1.1

### Repro Steps:​

1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Codespace"
3. Click on "Visual Studio codespace".
4. Navigate till "Remote Explorer" control.
5. Navigate on screen up to create new codespace and select plan button and observed that tool tip is present or not for both the buttons.

### Actual:
Tool tip is missing for create new codespace and select plan button on keyboard focus.

### Expected: ​
Tool tip should be given for create new codespace and select plan button on keyboard focus.

### ​User Impact: ​
Keyboard only user will face difficulty to understand the content without tooltip
